code/modules/mob/living/carbon/human/human_movement.dm

```diff
@@ -52,3 +52,17 @@
 	if(dna.species.space_move(src))
 		return TRUE
 	return ..()
+
+/mob/living/carbon/human/CanAllowThrough(atom/movable/mover, turf/target)
+	. = ..()
+	if(.) // If we're already letting them through, then might as well not check anything else.
+		return
```

Comment:
I am pretty sure an empty `return` will return a null; Test this please.

Reply:
Won't return null due to `. = ..()` which sets the return value `.` to the parent value `..()`

Reply:
Empty return returns `.` by default